src/components/Thanks/ShareChallenge.vue

```diff
@@ -209,7 +209,11 @@ export default {
 				team: this.teamPublicId,
 				lender: this.lender?.publicId ? `${this.lender.publicId}` : '',
 			};
-			return getFullUrl(`${base}/lend/filter`, args);
+			let invitedBy = '';
+			if (this.loan.id && this.lender?.inviterName) {
```

Comment:
Just to check, are these two values defined like expected for this challenge header?